torch/__init__.py

```diff
@@ -178,14 +178,22 @@ def _load_dll_libraries() -> None:
         else:
             base_py_dll_path = ""
 
+        # when using pip install on a project that depends on PyTorch, pip installs
+        # torch & mkl in a custom path that's included in sys.path but does not match
+        # any of the other DLLs path above, so we need to manually add it here.
+        mkl_dll_path = os.path.join(
+            os.path.dirname(__file__), "..", "..", "..", "Library", "bin"
```

Comment:
Where is the path? I s there a more stable way to find it?

Reply:
I don't think so, since the issue is that MKL is NOT in the PATH to begin with. 

Looking at this code again, I think I made a copy/paste mistake though, this should read `os.path.dirname(wheel.__file__), "..", ".."`: MKL gets installed in `<tmp-pip-prefix>/Library/bin`, and the only way I found to get this `<tmp-pip-prefix>` was to look where other build-time dependency such as `wheel` are installed.

I'll find one and update the code, sorry about that.